### PR TITLE
Delete user

### PR DESCRIPTION
**Pull request by simionescuraluca**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -3,3 +3,4 @@ spring.datasource.url=jdbc:mysql://localhost:3306/vinylstore
 spring.datasource.username=root
 spring.datasource.password=Za2s3Tcvas
 spring.jpa.database = MYSQL
+
```
Comment (dragosslujeru) [inline src/main/resources/application.properties]: unintended change

File: src/main/java/com/vinyl/repository/UserRepository.java [hunk 2]
```diff
@@ -8,5 +10,6 @@
 @Repository("userRepository")
 public interface UserRepository extends CrudRepository<User, Integer> {
 
-	User findByEmail(String email);
+	Optional<User> findByEmail(String email);
+	User findByPass(String password);
```
Comment (dragosslujeru) [inline src/main/java/com/vinyl/repository/UserRepository.java]: please delete this method as it is not used

File: src/main/resources/application.properties (hunk 1 above)
Comment (simionescuraluca) [reply src/main/resources/application.properties]: fixed

File: src/main/java/com/vinyl/repository/UserRepository.java (hunk 2 above)
Comment (simionescuraluca) [reply src/main/java/com/vinyl/repository/UserRepository.java]: fixed

File: src/main/java/com/vinyl/model/ProductCart.java [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 import java.io.Serializable;
 
+import javax.persistence.CascadeType;
```
Comment (dragosslujeru) [inline src/main/java/com/vinyl/model/ProductCart.java]: unused import

Comment (simionescuraluca) [reply src/main/java/com/vinyl/model/ProductCart.java]: fixed

**Review by dragosslujeru — APPROVED** [review]

**Pull request merged**